src/components/Cart/User/SignUp.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="np-sign-in">
+    <h2 class="np-sign-in__title">{{ $t("signUp") }}</h2>
+    <VForm :validation-schema="schema">
+      <div class="field-container">
+        <VField name="name" placeholder="Name" />
+        <ErrorMessage name="name" />
+      </div>
+      <div class="field-container">
+        <VField name="email" placeholder="E-mail" />
+        <ErrorMessage name="email" />
+      </div>
+      <div class="field-container">
+        <VField name="password" type="password" placeholder="Password" />
+        <ErrorMessage name="password" />
+      </div>
+      <div class="field-container">
+        <VField name="confirmPassword" type="password" placeholder="Password" />
+        <ErrorMessage name="confirmPassword" />
+      </div>
+      <div class="np-sign-in__container">
+        <button class="np-sign-in__button" type="submit">
+          {{ $t("signUp") }}
+        </button>
+
+        <button @click.prevent="goBack()" class="np-sign-in__button">
+          {{ $t("return") }}
+        </button>
+
+        <p class="np-sign-in__text">
+          {{ $t("signInMessage") }}
+          <strong>
+            <router-link to="/signIn">{{ " " + $t("signIn") }}</router-link>
```

Comment:
use {name: 'SignUp'} as "to" attribute, not path as path sometimes changes and name doesnt